src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgb(0, 0, 0, 0.05);
+
+  &__logo-link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+
+    &-list {
+      display: flex;
+      padding: 0;
+      gap: 20px;
+    }
+
+    &-item {
+      display: flex;
+      list-style: none;
```

Comment:
I think you have already reset this style in your file `reset.scss`, so here you can delete it